### DISCUSSION TITLE
SSL certificate and the Share preview link

### DISCUSSION THREAD

**Discussion by Root-ish** [barrel/shopify-vite#94] [Q&A]
Hi, I was wondering if someone can help me figure out how I can get the share preview link to work and view the website through  https?
When I'm running the Vite Shopify Example I get errors saying the certificate authority is invalid (`net::ERR_CERT_AUTHORITY_INVALID`). And when I remove the `server.https: true` line I still get an `net::ERR_EMPTY_RESPONSE` error. In my debugging quest I also removed the `basicSsl()` line which allows me to view the website locally but not through a share preview link. I'm a bit lost on what the problem could be.
I've also tried to use the `vite-plugin-mkcert` plugin instead of the basicSSL plugin but without any luck.

**Comment by nelsonvassalo** [comment]
I'm also curious as to how to preview the theme via the share link.
